tests/cli_commands/test_update.py

```diff
@@ -0,0 +1,30 @@
+import pathlib
+import unittest
+from unittest.mock import patch
+
+from click.testing import CliRunner
+
+from data_pipelines_cli.cli import _cli
+
+
+class UpdateCommandTestCase(unittest.TestCase):
+    copier_dst_path = "destination_path"
+    copier_vcs_ref = "HEAD"
+    copier_force = True
+    goldens_dir_path = pathlib.Path(__file__).parent.parent.joinpath("goldens")
+
+    def _mock_copier(self, dst_path: str, vcs_ref: str, force: bool):
+        self.assertEqual(self.copier_dst_path, dst_path)
+        self.assertEqual(self.copier_vcs_ref, vcs_ref)
+        self.assertEqual(self.copier_force, force)
```

Comment:
Unfortunately, `unittest` gets confused by those "nested" asserts 😞 and the errors it shows on failure appear very vague. It is better to set "global" variables here and then check them in the "right" test functions (cf. [[1](https://github.com/getindata/data-pipelines-cli/blob/develop/tests/cli_commands/test_run_test.py#L20), [2](https://github.com/getindata/data-pipelines-cli/blob/develop/tests/cli_commands/test_run_test.py#L46)]).

Edit: I see where it comes from (`test_create.py`), it should be fixed there too (but maybe sometime in the future).